### DISCUSSION TITLE
[FEATURE REQ] More freeleech filter options

### DISCUSSION THREAD

**Discussion by Slikkster** [autobrr/autobrr#1656] [Ideas]
**Is your feature request related to a problem? Please describe.**
One of the private trackers I use has torrents that are 2x free, i.e. free download and double upload. The current filter options do not have a way to select only these types of torrents. Just toggling on the freeleech option is not sufficient because there are also a large number of standard freeleech torrents that I don't want to snatch.

**Describe the solution you'd like**
A way to specify only 2x free torrents in the filter. Presumably the uploadvolumefactor attribute is available in the Torznab feed from jackett (this tracker does not have IRC announce so can't comment on that), so it would be possible to filter on uploadvolumefactor >= 2.

**Describe alternatives you've considered**
I am currently using the RSS feed from jackett for this tracker, and filtering it with Huginn to select only 2x free torrents for download (at which point I just have qbittorrent getting the filtered feed directly, no need for autobrr).

**Additional context**
Just discovered this app today and started setting it up, I think it's a great idea and excellent work so far. Looking forward to using it, thank you very much.

**Comment by Slikkster** [comment]
Additionally the filter for freeleech options is a bit confusing, I did not realize at first that the freeleech toggle and freeleech percentage fields were an either/or selection. Perhaps this could be made more clear by disabling the percentage field if freeleech is toggled on, with it prefilled to 100.

With the way it is now, it is possible to toggle freeleech on and also put 100 in the text field, which causes the filter not to match freeleech torrents at all.

**Comment by Audionut** [comment]
If it's the site I'm thinking of, check their RSS feeds. They have "Latest Featured" and "Latest Double Upload".

**Comment by zze0s** [comment]
I made some comments on the subject here. https://github.com/autobrr/autobrr/pull/678#issuecomment-1407437689

**Comment by Slikkster** [comment]
> If it's the site I'm thinking of, check their RSS feeds. They have "Latest Featured" and "Latest Double Upload".

It is not the site you are thinking of unfortunately, there are no options or feeds for freeleech/double upload. That is why I resorted to filtering the RSS feed with Huginn, because I'm not able to with autobrr.

**Comment by Slikkster** [comment]
> I made some comments on the subject here. [#678 (comment)](https://github.com/autobrr/autobrr/pull/678#issuecomment-1407437689)

Brilliant, sounds like you have already put quite a bit of thought into it, and already have a much better replacement system under way. Thank you for your work!
